.github/workflows/release.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+name: Release Teleporter
+
+on:
+  push:
+    tags:
+      - "v*.*.0"
+
+jobs:
+  build_and_upload_artifacts:
+    name: Build and Upload Teleporter Artifacts
+    runs-on: ubuntu-20.04
+    env:
+      deployment_tx_fn: TeleporterMessenger_Deployment_Transaction_${{ github.ref_name }}.txt
+      deployer_addr_fn: TeleporterMessenger_Deployer_Address_${{ github.ref_name }}.txt
+      contract_addr_fn: TeleporterMessenger_Contract_Address_${{ github.ref_name }}.txt
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      
+      - name: Set Go version
+        run: |
+          source ./scripts/versions.sh
+          echo GO_VERSION=$GO_VERSION >> $GITHUB_ENV
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+            go-version: ${{ env.GO_VERSION }}
+
+      - name: Install Foundry
+        run: ./scripts/install_foundry.sh
+    
+      - name: Build Contracts
+        run: |
+          export PATH=$PATH:$HOME/.foundry/bin
+          cd contracts/
+          forge build
+
+      - name: Build changelog
+        id: build_changelog
+        uses: mikepenz/release-changelog-builder-action@v4
+        with:
+          failOnError: true
+          configuration: "./.github/changelog.json"
+          toTag: ${{ github.ref_name }}
+
+      - name: Create Artifacts
+        id: artifacts
+        run: |
+          go run utils/contract-deployment/contractDeploymentTools.go constructKeylessTx contracts/out/TeleporterMessenger.sol/TeleporterMessenger.json
+          mv UniversalTeleporterDeployerTransaction.txt ${{ env.deployment_tx_fn }}
+          mv UniversalTeleporterDeployerAddress.txt ${{ env.deployer_addr_fn }}
+          mv UniversalTeleporterMessengerContractAddress.txt ${{ env.contract_addr_fn }}
+
+      - name: Create release
```

Comment:
Is the zipped source code automatically included, even though it's not specified explicitly? That's the default behavior when creating a release through Github, so it would be good to include that in the automated flow as well.

Reply:
Yes it's already included automatically